src/index.ts

```diff
@@ -28,40 +28,52 @@
 //     unmarshalled from an incoming request.
 
 import * as minimist from 'minimist';
+
 import {
   ErrorHandler,
   FunctionSignatureType,
   getServer,
   getUserFunction,
 } from './invoker';
 
+// Supported command-line flags
+const FLAG = {
+  PORT: 'port',
+  TARGET: 'target',
+  SIGNATURE_TYPE: 'signature-type', // dash
+};
+
+// Supported environment variables
+const ENV = {
+  PORT: 'PORT',
+  TARGET: 'TARGET',
```

Comment:
Perhaps I wasn't clear in my previous comments: we should keep the environment variable names unchanged, just like in the table in README. Please keep PORT, FUNCTION_TARGET, FUNCTION_SIGNATURE_TYPE.

Short names are fine for flag names and const names in the code.

Reply:
I mean:

```
const ENV = {
  PORT: 'PORT',
  TARGET: 'FUNCTION_TARGET',
  SIGNATURE_TYPE: 'FUNCTION_SIGNATURE_TYPE', // underscore
};
```

Reply:
Yes. Agreed.
Not sure why I changed this. Reverted the `FUNCTION_` prefix for these env vars.